src/patternfly/components/Dropdown/dropdown-multi-select.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="pf-c-dropdown__menu" role="dialog" aria-labelledby="{{id}}" {{#unless pf-c-dropdown--IsExpanded}}hidden="hidden"{{/unless}}>
+    <form class="pf-c-form">
+    <fieldset class="pf-c-form__fieldset">
+        <label class="pf-c-form__label  pf-c-dropdown__menu-item" for="checkbox-apples">
+            <input class="pf-c-check" type="checkbox" name="apples" id="checkbox-apples" checked> Apples
+        </label>
+        <label class="pf-c-form__label pf-c-dropdown__menu-item" for="checkbox-oranges">
```

Comment:
I'm not sure how much this matters at this point but this is not using the `pf-c-check` component properly. If you take a look you'll see you don't need to pull in a a form label b/c the component comes with the label. Maybe this PR was started before that change was merged.